src/main/java/seedu/address/model/assignment/DeadlineContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.assignment;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Assignment}'s {@code Deadline} matches any of the keywords given.
+ */
+public class DeadlineContainsKeywordsPredicate implements Predicate<Assignment> {
```

Comment:
Should be "Tests that **an** Assignment..." instead of "Tests that **a** Assignment...". Possible typo?

Reply:
Amended! Likewise for the other classes